.github/workflows/windows.yaml

```diff
@@ -0,0 +1,54 @@
+name: windows
+on:
+  push:
+    branches: [main]
+    tags: ['v*']
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron: '15 22 * * *'
+  workflow_dispatch: {} # support manual runs
+# Prevent writing to the repository using the CI token.
+# Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
+permissions: read-all
+jobs:
+  ci:
+    runs-on: windows-2019
```

Comment:
Can we use `windows-latest` instead? Outside the CLI and core, I'd like to minimize the number of places we need to bump version numbers. (Plus, I don't think any of us are tracking Windows server releases carefully.)